scripts/cut_release_branch.py

```diff
@@ -0,0 +1,189 @@
+
+PARSED_ARGS = _PARSER.parse_args()
+if PARSED_ARGS.new_version:
+    TARGET_VERSION = PARSED_ARGS.new_version
+else:
+    raise Exception('ERROR: A \'new_version\' arg must be specified.')
+
+# Construct the new branch name.
+NEW_BRANCH_NAME = 'release-%s' % TARGET_VERSION
+NEW_APP_YAML_VERSION = TARGET_VERSION.replace('-', '.')
+
+
+def _get_remote_alias():
+    # Find the correct alias for the remote branch.
+    git_remote_output = subprocess.check_output(
+        ['git', 'remote', '-v']).split('\n')
+    remote_alias = None
+    for line in git_remote_output:
+        if 'https://github.com/oppia/oppia' in line:
+            remote_alias = line.split()[0]
+    if remote_alias is None:
+        raise Exception(
+            'ERROR: There is no existing remote alias for the Oppia repo.')
+
+    return remote_alias
+
+
+def _verify_local_repo_is_clean():
+    """Checks that the local Git repo is clean."""
+    git_status_output = subprocess.check_output(
+        ['git', 'status']).strip().split('\n')
+    branch_is_clean = (
+        git_status_output[1] == 'nothing to commit, working directory clean')
+    if len(git_status_output) > 2 or not branch_is_clean:
+        raise Exception(
+            'ERROR: This script should be run from a clean branch.')
+
+
+def _verify_current_branch_is_develop():
+    """Checks that the user is on the develop branch."""
+    git_status_output = subprocess.check_output(
+        ['git', 'status']).strip().split('\n')
+    if git_status_output[0] != 'On branch develop':
+        raise Exception(
+            'ERROR: This script can only be run from the "develop" branch.')
+
+
+def _verify_target_branch_does_not_already_exist(remote_alias):
+    """Checks that the new release branch doesn't already exist locally or
+    remotely.
+    """
+
+    git_branch_output = subprocess.check_output(['git', 'branch'])
+    if NEW_BRANCH_NAME in git_branch_output:
+        raise Exception(
+            'ERROR: The target branch name already exists locally. '
+            'Run "git branch -D %s" to delete it.' % NEW_BRANCH_NAME)
+    git_ls_remote_output = subprocess.check_output(
+        ['git', 'ls-remote', '--heads', remote_alias])
+    remote_branch_ref = 'refs/heads/%s' % NEW_BRANCH_NAME
+    if remote_branch_ref in git_ls_remote_output:
+        raise Exception(
+            'ERROR: The target branch name already exists on the remote repo.')
+
+
+def _verify_target_version_is_consistent_with_latest_released_version():
+    """Checks that the target version is consistent with the latest released
+    version on GitHub.
+    """
+    response = urllib.urlopen(
+        'https://api.github.com/repos/oppia/oppia/releases/latest')
+    if response.getcode() != 200:
+        raise Exception(
+            'ERROR: Failed to fetch latest release info from GitHub')
+
+    data = json.load(response)
+    latest_release_tag_name = data['tag_name']  # Expect v2.5.4
```

Comment:
Is this comment right?

Reply:
Whoops! Nope, it's not, I've removed it. Thanks for catching this!